macros/get_bucket.sql

```diff
@@ -0,0 +1,23 @@
+{% macro get_bucket(field, output_field) -%}
```

Comment:
Small nitpick: I think it'd be nice to leave the `as column_name` in the model definition rather than passing `output_field` to the macro here, but I don't have a strong opinion.